code/modules/awaymissions/mission_code/capture_the_flag.dm

```diff
@@ -0,0 +1,283 @@
+	var/client/new_team_member = user.client
+	dust_old(user)
+	spawn_team_member(new_team_member)
+
+/obj/machinery/capture_the_flag/proc/dust_old(mob/user)
+	if(user.mind && user.mind.current && user.mind.current.z == src.z)
+		new /obj/item/ammo_box/magazine/wt550m9 (get_turf(user.mind.current))
+		new /obj/item/ammo_box/magazine/wt550m9 (get_turf(user.mind.current))
+		user.mind.current.dust()
+
+
+/obj/machinery/capture_the_flag/proc/spawn_team_member(client/new_team_member)
+	var/mob/living/carbon/human/M = new/mob/living/carbon/human(get_turf(src))
+	new_team_member.prefs.copy_to(M)
+	M.key = new_team_member.key
+	M.faction = team
+	M.equipOutfit(ctf_gear)
+
+/obj/machinery/capture_the_flag/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/weapon/twohanded/required/ctf))
+		var/obj/item/weapon/twohanded/required/ctf/flag = I
+		if(flag.team != src.team)
+			user.unEquip(flag)
+			flag.loc = get_turf(flag.reset)
+			points++
+			for(var/mob/M in player_list)
+				if(M.z == src.z)
+					M << "<span class='userdanger'>[user.real_name] has captured \the [flag], scoring a point for [team] team! They now have [points]/[points_to_win] points!</span>"
+		if(points >= points_to_win)
+			victory()
+
+
+
+/obj/machinery/capture_the_flag/proc/victory()
+	for(var/mob/M in mob_list)
+		if (M.z == src.z)
+			M << "<span class='narsie'>[team] team wins!</span>"
+			M << "<span class='userdanger'>The game has been reset! Teams have been cleared. The machines will be active again in 30 seconds.</span>"
+			M.dust()
+	for(var/obj/machinery/capture_the_flag/CTF in machines)
+		if(CTF.ctf_enabled == TRUE)
+			CTF.points = 0
+			CTF.ctf_enabled = FALSE
+			CTF.team_members = list()
+			spawn(300)
+				CTF.ctf_enabled = TRUE
+
+/obj/item/weapon/gun/projectile/automatic/pistol/deagle/CTF
+	desc = "This looks like it could really hurt in melee."
+	force = 75
+
+/obj/item/weapon/gun/projectile/automatic/wt550/CTF
+	desc = "This looks like it could really hurt in melee."
+	force = 50
+
+/datum/outfit/ctf
+	name = "CTF"
+	/obj/item/device/radio/headset
+	uniform = /obj/item/clothing/under/syndicate
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf
+	shoes = /obj/item/clothing/shoes/combat
+	gloves = /obj/item/clothing/gloves/combat
+	id = /obj/item/weapon/card/id/syndicate
+	belt = /obj/item/weapon/gun/projectile/automatic/pistol/deagle/CTF
+	l_pocket = /obj/item/ammo_box/magazine/wt550m9
+	r_pocket = /obj/item/ammo_box/magazine/wt550m9
+	r_hand = /obj/item/weapon/gun/projectile/automatic/wt550/CTF
+
+/datum/outfit/ctf/red
+	ears = /obj/item/device/radio/headset/syndicate/alt
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf/red
+
+/datum/outfit/ctf/blue
+	ears = /obj/item/device/radio/headset/headset_cent/commander
+	suit = /obj/item/clothing/suit/space/hardsuit/shielded/ctf/blue
+
+/datum/outfit/ctf/red/post_equip(mob/living/carbon/human/H)
+	var/obj/item/device/radio/R = H.ears
+	R.set_frequency(SYND_FREQ)
```

Comment:
Can't use this frequency because if the round is nuke flops, nuke flops need this.